src/languages/es.ts

```diff
@@ -2442,8 +2442,40 @@ export default {
                 },
                 importTaxDescription: 'Importar grupos de impuestos desde NetSuite',
                 importCustomFields: {
-                    customSegments: 'Segmentos/registros personalizados',
-                    customLists: 'Listas personalizado',
+                    customSegments: {
+                        title: 'Segmentos/registros personalizado',
```

Comment:
@yuwenmemon Can you verify the translations please?

Reply:
Plural form here
```suggestion
                        title: 'Segmentos/registros personalizados',
```

Custom Segments -> Segmentos personalizados 
Custom Records -> Registros personalizados
Custom Lists -> Listas personalizadas

That's the terminology from NetSuite.

Reply:
@zsgreenwald Can I enlist your help getting translations for these in #espanol-translations?